apply a small patch to dl-find callback.py for dlf_get_multistate_gradients_wrapper
https://raw.githubusercontent.com/Open-Quantum-Platform/openqp/5855118aa39304c1f8d175912de21b562387e40d/pyoqp/patch.sh
--- libdlfind/callback.py.orig	2025-03-26 15:48:17.000000000 +0000
+++ libdlfind/callback.py	2025-03-26 15:49:19.000000000 +0000
@@ -88,9 +88,11 @@
         needcoupling: int,
         iimage: int,
         status: pointer[c_int],
+        *args,
+        **kwargs,
     ) -> None:
         coordinates_ = as_array(coords, (nvar,)).reshape(-1, 3)
-        e_1, e_2, g_1, g_2, _ = func(coordinates_, needcoupling, iimage)
+        e_1, e_2, g_1, g_2, _ = func(coordinates_, needcoupling, iimage, *args, **kwargs)
         energy_ = as_array(energy, (2,))
         energy_[0] = e_1
         energy_[1] = e_2
